deps/amqp_client/src/amqp_channel.erl

```diff
@@ -872,11 +876,7 @@ do(Method, Content, Flow, #state{driver = direct, writer = W}) ->
 
 
 flush_writer(#state{driver = network, writer = Writer}) ->
-    try
-        rabbit_writer:flush(Writer)
-    catch
-        exit:noproc -> ok
-    end;
+    _ = catch rabbit_writer:flush(Writer);
```

Comment:
This won't always return `ok` anymore, is that intended? Before it was either returning `ok` or triggering an exception. Now it'll return `ok` or `{'EXIT', ...}` or other fancy tuples `catch` returns.

Reply:
You're right, let me fix that.

Reply:
Done!